lib/printer.js

```diff
@@ -1,8 +1,8 @@
 var printer_helper = {}
-    fs = require("fs")
-    child_process = require("child_process")
-    os = require("os")
-    path = require("path"),
+fs = require("fs")
+child_process = require("child_process")
+os = require("os")
```

Comment:
I haven't modified that code, I am traveling now and not available to insert right away, will try doing that a little later.